manifests/0000_30_cluster-baremetal-operator_01_images.configmap.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+        name: cluster-baremetal-operator-images
```

Comment:
Sorry one last thing. Can we fix the formatting here.

Reply:
Not sure how I missed that!